docs/Formula-Cookbook.md

```diff
@@ -696,6 +696,16 @@ Homebrew provides two formula DSL methods for launchd plist files:
 * [`plist_name`](https://www.rubydoc.info/github/Homebrew/brew/master/Formula#plist_name-instance_method) will return e.g. `homebrew.mxcl.<formula>`
 * [`plist_path`](https://www.rubydoc.info/github/Homebrew/brew/master/Formula#plist_path-instance_method) will return e.g. `/usr/local/Cellar/foo/0.1/homebrew.mxcl.foo.plist`
 
+### Using environment variables
+
+Homebrew has multiple levels of environment variable filtering which impact which variables are available to formulae.
+
+Firstly, the overall environment in which Homebrew runs is [filtered](https://github.com/Homebrew/brew/blob/master/bin/brew#L54-L96) to avoid environment contamination breaking from-source builds ([ref](https://github.com/Homebrew/brew/issues/932)).  In particular, this process filters all but the given whitelisted variables relating to the shell, terminal, OS, proxies and so on, but allows anything prefixed with `HOMEBREW_` or related to CI and Travis through.
```

Comment:
Can you remove `relating to the shell, terminal, OS, proxies and so on,` and `or related to CI and Travis through`; this is implied by looking at the code and if we update the code we'll likely forget to update these docs. 

Additionally, it may be worth noting the `#L54-L96` will break if the number of lines in that file changes. It may be worth omitting the lines count as we want to link to something up-to-date.

Reply:
For sure, I'll speak more generally and link to the general file itself so people can always get the right info.